src/appleseed/renderer/modeling/aov/depthaov.cpp

```diff
@@ -144,7 +143,7 @@ namespace
 
         void clear_image() override
         {
-            m_image->clear(Color<float, 1>(numeric_limits<float>::max()));
+            m_image->clear(Color<float, 1>(0.0f));
```

Comment:
You need to use Color<float, 2> since the image has two channels, and pass two values to it.